d3a_interface/settings_validators.py

```diff
@@ -37,54 +42,53 @@ def validate_global_settings(settings_dict):
         slot_length = (duration(minutes=slot_length.seconds / 60)
                        if isinstance(slot_length, timedelta) else duration(minutes=slot_length))
 
-    if "tick_length" in settings_dict:
+    if "tick_length" in settings:
         min_tick_length, max_tick_length = calc_min_max_tick_length(slot_length)
         if not min_tick_length <= tick_length <= max_tick_length:
             raise D3ASettingsException("Invalid tick_length "
                                        f"({tick_length.in_seconds()} sec, limits: "
                                        f"[{min_tick_length.in_seconds()} sec, "
                                        f"{max_tick_length.in_seconds()} sec])")
-    if "slot_length" in settings_dict:
+    if "slot_length" in settings:
         min_slot_length, max_slot_length = calc_min_max_slot_length(tick_length)
         if not min_slot_length <= slot_length <= max_slot_length:
             raise D3ASettingsException("Invalid slot_length "
                                        f"({slot_length.in_minutes()} min, limits: "
                                        f"[{min_slot_length.in_minutes()} min, "
                                        f"{max_slot_length.in_minutes()} min])")
-    if ("cloud_coverage" in settings_dict and
-        not (ConstSettings.PVSettings.CLOUD_COVERAGE_LIMIT.min
-             <= settings_dict["cloud_coverage"] <=
-             ConstSettings.PVSettings.CLOUD_COVERAGE_LIMIT.max)):
```

Comment:
Whats the reason to replace `min/max` with indices?

Reply:
@Hassan754 I have the same question. I saw that you reacted with a 👍  but the code did not seem to change.

Reply:
weird, github fails me again. I see the commits, but i dont see it in the global diff.
Ignore my comment.